### PR TITLE
linkedin's pygradle produces pex files

### PR DESCRIPTION
**Pull request by sixninetynine**
can we add a plug?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ include multiple platform-specific Python distributions, meaning that a single p
 can be portable across Linux and OS X.
 
 pex files can be built using the ``pex`` tool.  Build systems such as `Pants
-<http://pantsbuild.github.io/>`_ and `Buck <http://facebook.github.io/buck/>`_ also
+<http://pantsbuild.github.io/>`_, `{py}gradle <https://github.com/linkedin/pygradle>`_ and `Buck <http://facebook.github.io/buck/>`_ also
```
Comment (kwlzn) [inline README.rst]: since Buck technically adopted pex before pygradle, perhaps the ordering of mention should be Pants->Buck->{py}gradle?

Comment (sixninetynine) [inline README.rst]: fine by me! updated

**Comment by sixninetynine** [comment]
just a friendly ping :)

**Comment by kwlzn** [comment]
merged - thanks!

**Pull request merged**